webpack/components/AnsibleHostDetail/AnsibleHostDetail.js

```diff
@@ -40,6 +41,7 @@ AnsibleHostDetail.propTypes = {
 
 AnsibleHostDetail.defaultProps = {
   location: { pathname: '' },
+  router: PropTypes.object.isRequired,
```

Comment:
this should probably be in `propTypes`, which would also fix the complaining linter